app/src/main/java/com/github/se/polyfit/viewmodel/meal/MealViewModel.kt

```diff
@@ -41,6 +41,10 @@ class MealViewModel(
     }
   }
 
+  fun setMealName(name: String) {
```

Comment:
please rebase and add tests for this function :)

Reply:
done